src/tests/Common/GenerateHWIntrinsicTests/GenerateHWIntrinsicTests_Arm.cs

```diff
@@ -80,7 +80,7 @@
 
 const string SimpleVecOpTest_ValidationLogicForCndSelForNarrowing = @"for (var i = 0; i < Op1ElementCount; i++)
             {
-                {Op1BaseType} iterResult = (mask[i] != 0) ? {GetIterResult} : falseVal[i];
+                {RetBaseType} iterResult = (mask[i] != 0) ? {GetIterResult} : falseVal[i];
```

Comment:
Is this needed too?

Reply:
```suggestion
                {Op1BaseType} iterResult = (mask[i] != 0) ? {GetIterResult} : falseVal[i];
```

Reply:
fixed it.

Reply:
I'm going to test this version one more time. Not sure why these changes even appeared, and I think they were supposed to be part of the `ConvertToInt32` commit. These changes may be necessary for the tests to run properly.